README.md

```diff
@@ -246,6 +246,26 @@ for example. See [Creating & using secrets]. While still secure,
 [trusted publishing] is now encouraged over API tokens as a best practice
 on supported platforms (like GitHub).
 
+### Generating and uploading attestations (EXPERIMENTAL)
+
+> [!NOTE]
```

Comment:
Should this also mention the need to have Trusted publishing enabled? Perhaps, link to it?

Also, I'd put this section closer to the top of the document (right after trusted publishing is introduced, maybe?).